zio-http/src/main/scala/zhttp/service/server/ServerSocketHandler.scala

```diff
@@ -10,12 +10,12 @@ import zhttp.socket.{Socket, WebSocketFrame}
 final case class ServerSocketHandler[R, E](
   zExec: UnsafeChannelExecutor[R],
   socket: Socket[R, E, WebSocketFrame, WebSocketFrame],
-) extends JSimpleChannelInboundHandler[JWebSocketFrame] {
+) extends JSimpleChannelInboundHandler[JWebSocketFrame]
+    with ServerHttpExceptionHandler { self =>
```

Comment:
Can we rename `ServerHttpExceptionHandler` to `ServerExceptionHandler` since we are using it in Http as well as websocket handler